### PR TITLE
Test time aug predictions

### PR DESCRIPTION
**Pull request by tomer-algo**
# Intro
  - Add Test-Time Augmentation to inference (improve by ~2% Mean@0.1 metrics)
# Project
  - Suggested Skeleton
# Implemented Sections
  - Added to 
    - offline predict
    - export model
  - Did it require refactoring in other locations as well? No
# Tests Description
  - 
# Undeveloped Sections
  - 
# What's New
  - Add Test-Time Augmentation to inference
# Checklist
  - [x] Module tested fully after last commit
  - [ ] Scheduled a PR meeting (if needed)
  - [ ] Documentation (comments or a dedicated documnet)
### Tested platforms
  - [x] Local
  - [ ] Cloud (Sagemaker)
  - [ ] Staging
  - [ ] Not relevant
### Tested Cameras
  - [ ] Gen 1.5
  - [ ] Gen 2
  - [x] Not relevant
# Comments
### Included Features
  - https://github.com/nanit/deep-high-resolution-net.pytorch/pull/21
### Infrastructure Dependencies
  - 
### Tech Comments
  - 
### Testing Instructions
  - 
### Knowledge
| / | Link |
| ------ | ------ |
| Monday | https://nanit.monday.com/boards/912051282/pulses/1852251365?userId=17446237 |
| TLD |  |
| Tech Spec |  |
### Todos
  -

### CODE REVIEW

File: lib/core/inference.py [hunk 1]
```diff
@@ -107,21 +111,34 @@ def get_pose_estimation_prediction(pose_model, device, cfg, images, centers, sca
...
-        model_inputs.append(model_input)
+        model_inputs.append(transform(model_input))
 
     # n * 1chw -> nchw
     model_inputs = torch.stack(model_inputs)
     model_inputs = model_inputs.to(device)
 
+    model_inputs_flipped = output_flipped = None
+    if cfg.TEST.FLIP_TEST and flip_pairs:
+        model_inputs_flipped = model_inputs.flip(3)
```
Comment (Nimrods) [inline lib/core/inference.py:122]: what is "3"?

Comment (tomer-algo) [reply lib/core/inference.py:122]: width dim, i'll add a comment

**Review by Nimrods — APPROVED** [review]
